src/integrationTest/resources/application-test.yml

```diff
@@ -1,7 +1,13 @@
 application:
   queues:
+    account-confirmed: dummy
     placement-synced: dummy
     programme-membership-synced: dummy
+  sns:
+    arn: dummy
+
+mongock:
+  enabled: false
```

Comment:
The integration tests were broken (for me) on the main branch because of mongock bean issues 🤷 .